lib/Client/ClientTracer.php

```diff
@@ -282,7 +284,7 @@ public function join($operationName, $format, $carrier) {
             break;
 
         case LIGHTSTEP_FORMAT_BINARY:
-            throw new Exception('FORMAT_BINARY not yet implemented');
+            throw new \Exception('FORMAT_BINARY not yet implemented');
```

Comment:
if using namespaces, better practice to include the `\` for classes like `Exception`